tests/files/no-unused-modules/main/index.js

```diff
@@ -0,0 +1 @@
+export const main = 'main'
```

Comment:
please ensure every file has a trailing newline

Reply:
I always miss that. In my other projects it gets added automatically by ESLint/Prettier...
Can we probably add `eol-last` to the eslint-config of this project?

Reply:
Sure, sounds great.

Reply:
(but, your editor should add a trailing newline by default in every file, entirely unrelated to eslint)